tests/tests_pytorch/strategies/test_ddp_fully_sharded_native.py

```diff
@@ -148,6 +148,7 @@ def custom_auto_wrap_policy(
     return unwrapped_params >= 2
 
 
+# lite: adopted
```

Comment:
Show reviewers that we adopted all tests that are in PL. 
TODO: remove before merge